src/scenes/Projects/Files/ProjectFilesList.tsx

```diff
@@ -98,38 +95,38 @@ const useStyles = makeStyles(({ palette, spacing }) => ({
   },
 }));
 
-export const ProjectFilesList: FC = () => {
+type ProjectDirectoryFileNode = ProjectDirectoryQuery['directory']['children']['items'][0];
+export type ProjectDirectoryDirectory = Exclude<
+  ProjectDirectoryFileNode,
+  FileNodeInfo_FileVersion_Fragment | FileNodeInfo_File_Fragment
+>;
+export type ProjectDirectoryFile = Exclude<
+  ProjectDirectoryFileNode,
+  FileNodeInfo_Directory_Fragment | FileNodeInfo_FileVersion_Fragment
+>;
+export type ProjectDirectoryNonVersion = Exclude<
+  ProjectDirectoryFileNode,
+  FileNodeInfo_FileVersion_Fragment
+>;
```

Comment:
It might be easier to move create gql fragments so types are generated for you instead of having to slog through the query type.

Reply:
Ah, good catch!

Reply:
So much better